src/Obj.hs

```diff
@@ -1150,3 +1151,42 @@ walk :: (XObj -> XObj) -> XObj -> XObj
 walk f (XObj (Arr xs) i t) = XObj (Arr (map (walk f) xs)) i t
 walk f (XObj (Lst xs) i t) = XObj (Lst (map (walk f) xs)) i t
 walk f x = f x
+
+tyToObj :: Ty -> Obj
```

Comment:
Ideally, I think we could define this as:

```haskell
tyToObj = xobjObj . reify
```

And perform further transformations (if any are needed) downstream. (e.g. calling `simpleSym`)

Reply:
See `Reify.hs`

Reply:
We'd also have to move the definition of this function into `Reify` to define it this way (to avoid import cycles)—but that seems ok to me since the responsibility of `Reify` is to provide conversions from internal Type representations to AST things

Reply:
Actually, I just completely forgot about `reify`, and the usecase can be expressed completely in terms of it. This means I can get rid of this entirely! Thanks!

Reply:
Fixed in 04588784

Reply:
Nice! Thanks!